src/ploneorg/addonlisting/browser/templates/addonfolder_view.pt

```diff
@@ -1,40 +1,55 @@
-<html xmlns="http://www.w3.org/1999/xhtml"
-      xmlns:metal="http://xml.zope.org/namespaces/metal"
-      xmlns:tal="http://xml.zope.org/namespaces/tal"
-      xmlns:i18n="http://xml.zope.org/namespaces/i18n"
-      i18n:domain="ploneorg.addonlisting"
-      metal:use-macro="context/main_template/macros/master">
-<body>
-  <metal:content-core fill-slot="content-core">
-
-  <metal:addonlist define-macro="addonlist">
-  <table class="listing"
-         id="AddOns">
-    <thead>
-      <tr>
-        <th>Title</th>
-        <th>Description</th>
-      </tr>
-    </thead>
-    <tbody>
-      <tr tal:repeat="addon view/all_addons" tal:condition="view/all_addons">
-        <td tal:content="addon/title">
-            Title
-        </td>
-        <td tal:content="addon/description">
-            Description
-        </td>
-      </tr>
-      <tr tal:condition="not:view/all_addons">
-        <td colspan="2">
+<html
+  xmlns="http://www.w3.org/1999/xhtml"
+  xmlns:metal="http://xml.zope.org/namespaces/metal"
+  xmlns:tal="http://xml.zope.org/namespaces/tal"
+  xmlns:i18n="http://xml.zope.org/namespaces/i18n"
+  i18n:domain="ploneorg.addonlisting"
+  metal:use-macro="context/main_template/macros/master">
+  <head>
+    <metal:block fill-slot="style_slot">
```

Comment:
This should not be necessary at the end. Styling does belong to a larger scope and do not need to be explicit added per view